packages/thirdweb/src/indexer/indexer.ts

```diff
@@ -0,0 +1,24 @@
+export async function getIndexerClient(params, body) {
+  // TODO: store the url in a config file and change it based on the environment
+  const url = `http://${chainId}.indexer.thirdweb-dev.com`;
+
+  const indexerClient = (async () => {
+    try {
+      // TODO: add paging support
+      const response = await fetch(`${url}/${params}`, {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
```

Comment:
I'm assuming you are going to want some kind of auth here? if so this helper function should likely take in a `ThirdwebClient` which will have `clientId` and / or `secretKey` on it. There also already is `getClientFetch` which will give you a `fetch` function with all the auth headers etc set correctly across thirdweb services.

Reply:
Used `getClientFetch`